TCAT/Cells/SearchResultsCell.swift

```diff
@@ -39,7 +39,7 @@ class SearchResultsCell: UITableViewCell {
         imageView?.contentMode = .scaleAspectFit
         imageView?.center.y = bounds.height / 2.0
         imageView?.image = #imageLiteral(resourceName: "pin")
-        imageView?.tintColor = reuseIdentifier == "cornellDestinations" ? .tcatBlueColor : .placeColor
+        imageView?.tintColor = reuseIdentifier == Key.Cells.cornellDestinationsIdentifier ? .tcatBlueColor : .mediumGrayColor
```

Comment:
same comment. think resuseIdentifiers vars should be associated with the cell their in rather than a global Key struct

Reply:
It's used in multiple files though too